packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -395,6 +417,23 @@ const App = (props) => {
                         </Box>
                     </CurrencyProvider>
                 </IntlProvider>
+                {ACTIVE_DATA_ENABLED && (
+                    <script
+                        type="text/javascript"
+                        src={getAssetUrl('static/dwanalytics-22.2.js')}
+                        id="dwanalytics"
+                        async="async"
+                        onLoad={trackPage}
+                    ></script>
+                )}
+                {ACTIVE_DATA_ENABLED && (
+                    <script
+                        src={getAssetUrl('static/dwac-21.7.js')}
+                        type="text/javascript"
+                        id="dwac"
+                        async="async"
+                    ></script>
+                )}
```

Comment:
```suggestion
                <ActiveData />
```

To improve extensibility (e.g. if customers want to upgrade to some future version of the included scripts), I'd recommend extracting these to a component in a separate file (including the `head-active_data` script from above):

```jsx
// app/components/active-data/index.jsx
const ActiveData = () => {
    if (!ACTIVE_DATA_ENABLED) return
    return (
        <>
            <Helmet>
                <script
                    src={getAssetUrl('static/head-active_data.js')}
                    id="headActiveData"
                    type="text/javascript"
                ></script>
            </Helmet>
            <script
                type="text/javascript"
                src={getAssetUrl('static/dwanalytics-22.2.js')}
                id="dwanalytics"
                async="async"
            ></script>
            <script
                src={getAssetUrl('static/dwac-21.7.js')}
                type="text/javascript"
                id="dwac"
                async="async"
            ></script>
        </>
    )
}
```

Reply:
We already ruled out all those concerns with John and Brian. We decided to keep it as is for now.